CMakeLists.txt

```diff
@@ -108,9 +108,9 @@ install_basic_package_files(${PROJECT_NAME} VARS_PREFIX ${VARS_PREFIX}
 
 include(AddUninstallTarget)
 
-if(ENABLE_TESTS)
+if(YARPWHOLEBODYINTERFACE_ENABLE_TESTS)
 #enable testing
-include(CTest)
+#include(CTest)
```

Comment:
My bad, this line is required just for submitting test results to the dashboard. At the moment we are not using this feature, so this can be safely disabled.